View/ViewHandler.php

```diff
@@ -138,6 +141,8 @@ public function __construct(
         $this->serializeNull = $serializeNull;
         $this->forceRedirects = (array) $forceRedirects;
         $this->defaultEngine = $defaultEngine;
+        $this->options = $options;
```

Comment:
`$this->options = $options + ['exclusionStrategyGroups' => [], 'exclusionStrategyVersion' => null, 'serializeNullStrategy' => null];` to remove isset checks in `reset()` and ensure fields are reset to their default value if none was explicitely provided :)